test/core/models/ocelot/stanzas/ChoiceStanzaSpec.scala

```diff
@@ -144,7 +144,7 @@ class ChoiceStanzaSpec extends BaseSpec {
 
   "ChoiceStanza" must {
 
-    "deserialise from json" in {
+    "deserialize from json" in {
```

Comment:
American spelling ????

Reply:
Surprisingly, according to the Shorter Oxford Dictionary (1983) no. The correct spelling is serialize. I just changed the spelling because IntelliJ flagged up a mistake. Now that I am better informed I've updated the rest of the class.

It may be that the root is Greek not Latin like authorize.